languages/python/sqlalchemy-oso/sqlalchemy_oso/session.py

```diff
@@ -11,6 +11,29 @@
 from sqlalchemy_oso.auth import authorize_model_filter
 
 
+class OsoSession:
+    @classmethod
+    def get(cls):
+        return cls._get()
+
+    @classmethod
+    def set_get_session(cls, get_session):
+        OsoSession._get = get_session
+
+
+def set_get_session(oso: Oso, get_session_func):
+    """Set the function that oso uses to expose a SQLAlchemy session to the policy
+
+    :param [oso]: [The Oso instance used to evaluate the policy.]
```

Comment:
Is this doc string format correct? I haven't seen brackets in it before.

Reply:
Can we include in here how to access the session this creates from the policy? Just a polar snippet?

Reply:
idk this was a sphinx thing I think

Reply:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html